src/components/Proposal/Create/SchemeForms/CreateSchemeRegistrarProposal.tsx

```diff
@@ -471,6 +466,7 @@ class CreateSchemeRegistrarProposal extends React.Component<IProps, IState> {
                           placeholder="e.g. 0x0000000000000000000000000000000000000000000000000000000000001234"
                           name="parametersHash"
                           className={touched.parametersHash && errors.parametersHash ? css.error : null}
+                          validate={async () => { if (!(await arc.verifyParametersHash(this.props.scheme.address, values.parametersHash))) { return "Invalid parameters hash"; } }}
```

Comment:
1. Check validity of params hash. same as https://github.com/daostack/alchemy/pull/2313/files#diff-6ddd9609c50acf7519173ec4990436eb81e5063f22e6d576f9dd8f0fee28ca47L329 . and if not valid show : "Invalid parameters hash" msg.
2. check if parameters are set and if not ,show : " scheme parameters not set"